### ISSUE TITLE
Cache content files to avoid requesting more than once

### ISSUE THREAD

**Issue by agusaldasoro** [decentraland/unity-renderer#837]
When getting content files from the content server many of them are used in more than one place. For example base avatars or files from scenes.

We should cache every call to the catalyst to GET `/content/contents/<CONTENT_ID>` endpoint so we can avoid requesting the same content more than once. We currently identified more than 10 requests in a period of 5 minutes to the same content.

## How to reproduce this issue?
Enter into play.decentraland.org and watch the requests that are made to peer to the path `content/contents/<CONTENT_ID>`, there are many requests to that endpoint as that is the endpoint used to download files. Anyway, many files are in many different entities such as base avatars or scene images or audio.

## Examples
The following examples are two from the ones  captured during the first minutes after entering into Decentraland:

### Off-Chain Base Wearable

- URL: `https://peer-eu1.decentraland.org/content/contents/QmcDem7LfRM6R7WMxch7zPXhhVezETwoJRnq2yoVd74bYd`
- Wearable entity: [urn:decentraland:off-chain:base-avatars:basefemale](https://peer.decentraland.org/content/deployments?entityId=QmP6beggdEtiyzTBusHGvSBrk8Cc4ecYYiRT1DSfARMFna)

### Scenes files: sounds/click.mp3

- URL: `https://peer-eu1.decentraland.org/content/contents/QmapJcmjxssLhWEax9cz5i2X18uMTQzAjx1RDSevUYYayD`
- Scenes:  [Main Plaza scene](https://peer.decentraland.org/content/deployments?entityId=QmZZjR38Jen4MwisxDaUS3amgjfQLzsRKqiCxAWENJumW6) & [Other scenes with the same file](https://peer.decentraland.org/content/contents/QmapJcmjxssLhWEax9cz5i2X18uMTQzAjx1RDSevUYYayD/active-entities/)

### UPDATE 2021-09-01:
Looks like some images like profile snapshots or wearable thumbnails and/or textures (seems like mostly face related wearables) are not getting cached
https://docs.google.com/spreadsheets/d/1KZTORsE0ZJvDKIpMD_jbz7JBSvRfIMXMhxfZOZugZwU/edit#gid=220842179

**Comment by olavra** [comment]
@agusaldasoro We need more information to measure this. Can we have the hashes of the assets are being requested more often?

**Comment by agusaldasoro** [comment]
> @agusaldasoro We need more information to measure this. Can we have the hashes of the assets are being requested more often?

Sure! I'm updating the issue to reflect the examples.

**Comment by Suduck** [comment]
Pending on the changes that we are already doing from the memory leaks, after that changes this shouldn't happen anymore

**Comment by olavra** [comment]
@pbosio have you checked this after merging the AudioClips promise keeper?

**Comment by pbosio** [comment]
Updated issue:

> Looks like images some images like profile snapshots or wearable thumbnails and/or textures (seems like mostly face related wearables) are not getting cached
> https://docs.google.com/spreadsheets/d/1KZTORsE0ZJvDKIpMD_jbz7JBSvRfIMXMhxfZOZugZwU/edit#gid=220842179

**Comment by BrianAmadori** [comment]
#1158 is addressed to optimize the content server load. Through it does so by assuming the avatar snapshots will seldom be used instead of caching them. Requests will only be made if users open passports, add the users as friends and open the friends panel or open the neighboring users UI.

**Comment by olavra** [comment]
This is no longer needed as we do not need to worry about optimizing request to content server as much as before
